kubeflow/hub/api/model_registry_client_test.py

```diff
@@ -0,0 +1,144 @@
+"""Tests for ModelRegistryClient."""
+
+from __future__ import annotations
+
+from unittest.mock import Mock
+
+import pytest
+
+
+class TestModelRegistryClientInitialization:
```

Comment:
I'm happy to take a different approach to testing if preferred. I generally don't love mocking this much, though this is just a thin wrapper